app/schemas/interactions/nudged.avsc

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "interactions.nudged",
+  "type": "record",
+  "fields": [
```

Comment:
Do we need [a namespace](https://github.com/a15k/interactions-api/blob/41dba4ed0360d0247fbfa727422b368332b6c38a/app/schemas/org/a15k/flagged_content.avsc#L5)?

Reply:
@jpslav i'm not sure, i took a stab here...that's a good question for us to discuss.  i thought of this might be cool to organize schema definitions..but maybe not?  part of this was also just to test how namespaces/directories work....

Reply:
@reedstrm your opinion?

Reply:
"namespaces are good, let's do more of them" to paraphrase someone or other. Avoiding collisions is great.